modules/events_archive/EED_Events_Archive.module.php

```diff
@@ -598,146 +508,203 @@ public static function event_venues($content)
 
 
     /**
-     *    _add_additional_content_filters
-     *
-     * @access    private
-     * @return        void
+     * @return bool
+     * @since $VID:$
+     */
+    public static function isBlockTheme()
+    {
+        return function_exists('wp_is_block_theme') && wp_is_block_theme();
+    }
```

Comment:
@hoseinrafiei 
extracted this fix of your from a previous PR

Reply:
Great.
Actually, I was going to do this myself. My plan was to add this code in the `EEH_Template` class.

Reply:
oh ya, that would be even better, good thinking